featuremanagement/_featuremanager.py

```diff
@@ -8,7 +8,9 @@
 from ._featurefilters import FeatureFilter
 from ._models._feature_flag import FeatureFlag
```

Comment:
Could remove this since the new import includes it.

Reply:
Didn't notice that, that's why the other PR is needed.